src/components/GitGraph.tsx

```diff
@@ -25,90 +29,48 @@ function GitGraph() {
   }
 
   return (
-    <>
-    <h1 className="text-2xl font-bold text-center">{username}</h1>
-      <div className="block lg:hidden">
-        {commitData?.map((commit) => (
-          <div key={commit.id} className='border-t border-b border-gray-300 p-2 flex items-center mx-2'>
-          <img src="https://avatars.githubusercontent.com/u/90321356?v=4" alt="Avatar" className="h-8 w-8 rounded-full mr-2"></img>
-          <div className='w-full'>
-            <div className="flex justify-between gap-4 text-gray-400 w-full">
-              <p>{commit.author?.name ? commit.author.name : "Unknown"}</p>
-              <p>{commit.date}</p>
-              <p>{commit.id}</p>
-            </div>
-            <h2 className='italic'>{commit.message}</h2>
-            </div>
-          </div>
-        ))}
-      </div>
-      <div className="hidden lg:block">
-        <table className="bordered-table gap-4 w-screen border border-gray-300">
-          <thead className="w-full">
-            <tr className="border border-gray-300 column">
-              <th className="border border-gray-300 ">Description</th>
-              <th className="border border-gray-300 ">Date</th>
-              <th className="border border-gray-300 ">Author</th>
-              <th className="border border-gray-300 ">Commit</th>
-            </tr>
-          </thead>
-          <tbody>
-            {commitData?.map((commit) => (
-              <tr key={commit.id}>
-                <td className="pl-4">
-                  <ReadMore>{commit.message}</ReadMore>
-                </td>
-                {/* <td>{commit.message}</td> */}
-                <td className="text-center">{commit.date}</td>
-                <td className="text-center">{commit.author?.name ? commit.author.name : "Unknown"}</td>
-                <td className="text-center">{commit.id}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
-      </div>
-    </>
+    <></>
```

Comment:
Are you still planning to use this component? Just curious since it now just returns a fragment? I see that there is similar logic in GitTable.

Reply:
The GitGraph is for future use. I'm struggling with the logic, so I'm just leaving it till the end. I will remove it if I do not get the time for it